cwms-data-api/src/test/java/cwms/cda/api/OfficeControllerIT.java

```diff
@@ -160,5 +163,12 @@ enum AliasTest
             _expectedFormat = expectedFormat;
             _expectedGetAllStatusCode = expectedGetAllStatusCode;
         }
+/*
+        @Override
+        public String toString()
```

Comment:
Dead function

Reply:
OH thanks. not sure why the enum toString never got called for the parameterized test display name but I forgot to delete it.